pennylane/ops/channel.py

```diff
@@ -22,7 +23,7 @@
 
 
 class AmplitudeDamping(Channel):
-    r"""AmplitudeDamping(gamma, wires)
+    r"""AmplitudeDamping(gamma, wires, do_queue=True, id=None)
```

Comment:
@josh146, Is this what you mean by Sphinx Signature Override ?

Reply:
Yep 🙂 I suppose here it is serving two purposes:

- it is hiding `do_queue` and `id` (since we thought at the time it might be confusing to users)
- it is renaming `*params` to `gamma`.

Reply:
Cool so we can remove all instances of these after my changes !